### DISCUSSION TITLE
优化数据库管理

### DISCUSSION THREAD

**Discussion by superen7527** [1Panel-dev/1Panel#5905] [Q&A]
### 1Panel 版本

最新版

### 请描述您的需求或者改进建议

希望可以像宝塔那种一样 点击phpadmin之类管理直接跳转新页面，`然后自动链接到打开时对应的数据库`。

因为数据库一般不开外网，内网如果很多个数据库也不会映射，都是一套compose内网用。如果需要映射内网端口有点复杂，因为目前防火墙对docker兼容性问题，再加上phpadmin这种也是用docker安装，导致即使映射，有一些系统或者防火墙的问题会多层内网很难链接上，即使开放了防火墙对应端口。除非phpadmin用host，但是phpadmin改不了内置端口，默认监听80，就和nginx冲突了。用网段或者别的办法，其实有点麻烦，要是做成全自动链接就更好了。

不过要是太麻烦，也有其他方案，比如 添加phpadmin修改端口的配置，或者phpadmin安装到本机，不过最适合1p理念和容器管理化我感觉还是通过1p内部自动用桥ip段怎么实现内网链接，全自动化不用映射，点击按钮即用，这种方案比较好。

**Comment by wanghe-fit2cloud** [comment]
由于 1Panel 是容器化安装的应用，直接打开对应的数据库需要修改 phpMyAdmin 的环境变量，并重启服务。如果存在多个数据库，这可能会影响到其他服务的正常使用。因此，仍然需要用户手动填写数据库地址。
